theme/Colors.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  gray: '#5E6D8C',
```

Comment:
I would choose more generic names, like: "primary", "secondary", "text".

Reply:
And what in case if we would want to set the color of some icon to the same as the text? :smiley:

Reply:
Maybe should create a color named `icon`? :D And then I think in the module you can keep colors with explicit names, but probably you should export something more general, for example:
`const gray = '#5E6D8C';
export default {
  text: gray,
  icon: gray,
}
`
But it's also thing to discuss.